doc: exitedAfterDisconnect value can be false (#29404)
nodejs/node: doc/api/cluster.md

* {boolean}

> Set by calling `.kill()` or `.disconnect()`. Until then, it is `undefined` or `false`.

Reviewer: Can you add an explanation as to when it is `undefined` and when it is `false`?

Author: Yeah sure

> Set by calling `.kill()` or `.disconnect()`. Until the worker has not exited the value is `undefined`, if the worker has exited in any other way the value is `false`.

Reviewer: ```suggestion
This property is `true` if the worker exited due to `.kill()` or
`.disconnect()`. If the worker exited any other way, it is `false`. If the
worker has not exited, it is `undefined`.
```
